### PR TITLE
add templates to the base page and add register page 

### LINKED ISSUE #86: Create a registration page for a new user
## Expected Behavior
Create a registration page for a new user
## Steps to Reproduce the Problem

1.Using a user's directory
1.And creating the registration page
1.html to this page

**Please mention someone to take care of the issue**
avraham

### PR DESCRIPTION
**Pull request by avraham1992**
Fixes #86 

## Changes you made

- Adding a new user registration page including the registration page.
- Addition to the crispy project and a base page that includes a top bar and a flexible design
- **Please mention someone for code review**
Adding a new user registration page including the registration page. Addition to the crispy project and a base page that includes a top bar and a flexible design
@orzionpour Approve the design?

### CODE REVIEW

**Comment by orzionpour** [comment]
Avraham try the following flow:
1. Move to branch 'main'  - notice you don't delete your changes by mistake.
2. Create new branch
3. Cherry pick the commits you completed right now in this PR (using GitHub Desktop make it **a lot** easier).
4. Push the new branch to origin.
5. Create new PR.

File: users/forms.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+
+class UserRegisterForm(UserCreationForm):
+    email = forms.EmailField()
+
```
Comment (orzionpour) [inline users/forms.py:8]: You have a white space in this line - that's why the flake8 fails. delete it.
```suggestion

```

File: falcon/settings.py [hunk 2]
```diff
@@ -31,6 +31,7 @@
 INSTALLED_APPS = [
```
Comment (mgold1234) [inline falcon/settings.py:31]: unnecessary blank line

File: falcon/settings.py [hunk 3]
```diff
@@ -114,6 +115,9 @@
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
 STATIC_URL = '/static/'
```
Comment (mgold1234) [inline falcon/settings.py:117]: unnecessary blank line

File: templates/base.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
+{% load static %}
+<!DOCTYPE html>
+<html>
+<head>
+	 <!-- Required meta tags -->
```
Comment (mgold1234) [inline templates/base.html:5]: move it to the left
align it with next row

**Comment by mgold1234** [comment]
please fix flake8 error
./users/forms.py:8:1: W293 blank line contains whitespace
./users/forms.py:12:1: W293 blank line contains whitespace

**Pull request closed without merging**